nav2_smac_planner/CMakeLists.txt

```diff
@@ -92,6 +94,10 @@ add_library(${library_name} SHARED
 
 target_link_libraries(${library_name} ${OMPL_LIBRARIES} ${OpenMP_LIBRARIES}  OpenMP::OpenMP_CXX)
 target_include_directories(${library_name} PUBLIC ${Eigen3_INCLUDE_DIRS})
+if(WIN32)
```

Comment:
This should be set on both linux and windows, no need to wrap in WIN32

Reply:
fixed this